README.md

```diff
@@ -102,11 +103,20 @@ The ACAP Runtime service provides the following APIs:
 
 ### Installation
 
-Both variants of ACAP Runtime, the native ACAP application and the containerized
-version, are available as pre-built images on [Docker Hub][docker-hub-acap-runtime].
+The native ACAP Runtime application is available as a signed eap-file in [Releases][latest-releases]. The containerized version is available as a pre-built image on [Docker Hub][docker-hub-acap-runtime].
+
 These images are the recommended way to install and use ACAP Runtime.
 
-#### Native ACAP application
+#### **Native ACAP application for versions >1.2.0**
```

Comment:
Remove the *:s in the heading - should be handled by markdown

I think it would be good to mention that the eap file is signed and possibly add a link to documentation (e.g. https://axiscommunications.github.io/acap-documentation/docs/faq/security.html#sign-acap-applications)
Add the <=1.2.0 part as a subheading (e.g. ##### Installation of releases up untill 1.2.0). It is probably enough to only describe the part about installing. The parts about starting, stopping removing could be moved down to the Building ACAP Runtime section